src/main/steps/common/progress-bar.njk

```diff
@@ -35,14 +35,14 @@
     label: {
       html: textWithVisuallyHiddenSpan(submitted, displayState.isAtOrAfter('Submitted'))
     },
-    complete: displayState.isAtOrAfter('Submitted'),
-    active: displayState.at('AwaitingDocuments')
+    complete: displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'),
+    active: displayState.at('AwaitingDocuments') and not (displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'))
```

Comment:
Same question as [line 13](https://github.com/hmcts/nfdiv-frontend/pull/3884/files#r1768142405) - not sure we need the second part of the active condition?